### PR TITLE
Add setter for KIE_SERVER_ID to scenario builders

### PR DESCRIPTION
**Pull request by jhrcek**
@sutaakar the change you suggested. I also cleaned up the code a bit (remove unnecessary this., added final etc).

### CODE REVIEW

File: framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchKieServerDatabaseScenarioBuilderImpl.java [hunk 1]
```diff
@@ -52,4 +51,10 @@ public WorkbenchKieServerDatabaseScenarioBuilder withExternalMavenRepo(String re
         envVariables.put(OpenShiftTemplateConstants.MAVEN_REPO_PASSWORD, repoPassword);
         return this;
     }
+
+    @Override
+    public WorkbenchKieServerDatabaseScenarioBuilder withKieServerId(String kieServerId) {
+        envVariables.put(OpenShiftTemplateConstants.KIE_SERVER_ID, kieServerId);
+        return null;
```
Comment (sutaakar) [inline framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchKieServerDatabaseScenarioBuilderImpl.java]: Please return this to allow usage as a fluent API.

File: framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchKieServerScenarioBuilderImpl.java [hunk 2]
```diff
@@ -52,4 +51,10 @@ public WorkbenchKieServerScenarioBuilder withExternalMavenRepo(String repoUrl, S
         envVariables.put(OpenShiftTemplateConstants.MAVEN_REPO_PASSWORD, repoPassword);
         return this;
     }
+
+    @Override
+    public WorkbenchKieServerScenarioBuilder withKieServerId(String kieServerId) {
+        envVariables.put(OpenShiftTemplateConstants.KIE_SERVER_ID, kieServerId);
+        return null;
```
Comment (sutaakar) [inline framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchKieServerScenarioBuilderImpl.java]: Same as above

File: framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchRuntimeSmartRouterKieServerDatabaseScenarioBuilderImpl.java [hunk 3]
```diff
@@ -61,4 +60,10 @@ public WorkbenchRuntimeSmartRouterKieServerDatabaseScenarioBuilder withSmartRout
         envVariables.put(OpenShiftTemplateConstants.KIE_SERVER_ROUTER_ID, smartRouterId);
         return this;
     }
+
+    @Override
+    public WorkbenchRuntimeSmartRouterKieServerDatabaseScenarioBuilder withKieServerId(String kieServerId) {
+        envVariables.put(OpenShiftTemplateConstants.KIE_SERVER_ID, kieServerId);
+        return null;
```
Comment (sutaakar) [inline framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchRuntimeSmartRouterKieServerDatabaseScenarioBuilderImpl.java]: Same as above

**Comment by jhrcek** [comment]
@sutaakar I completely forgot about this PR. Comments addressed. Good catch, I rebased on master and fixed the `return null` problem.

**Comment by sutaakar** [comment]
retest this please

**Pull request merged**